src/main/java/by/exadel/internship/service/impl/UserServiceImpl.java

```diff
@@ -149,4 +153,14 @@ public void updateTimeSlot(UUID userId, List<UserTimeSlotDTO> userTimeSlotDTOLis
         return mapper.map(users);
     }
 
+    @Override
+    public Set<UserDTO> getUsersBySkills(List<Skill> skills) {
+        List<String> skillsToString = skills.stream().map(Enum::toString).collect(Collectors.toList());
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
```

Comment:
Looks like an extra line. There is no need to put class name in MDC.

Reply:
I dropped it